saleor/graphql/account/tests/test_account.py

```diff
@@ -4978,11 +4978,12 @@ def test_account_reset_password_user_is_inactive(
         "channel": channel_USD.slug,
     }
     response = user_api_client.post_graphql(REQUEST_PASSWORD_RESET_MUTATION, variables)
-    content = get_graphql_content(response)
-    data = content["data"]["requestPasswordReset"]
-    assert data["errors"] == [
-        {"field": "email", "message": "User with this email is inactive"}
-    ]
+    results = response.json()
+    assert "errors" in results
+    assert (
```

Comment:
Yes but that is not the case here as the error is one level higher. Please note that previously there was a `content = get_graphql_content(response)` call. It's not there anymore as it would crash due to assert within that function. That's why I changed this assert here.